### PR TITLE
Updating tools/cami_amber from version 2.0.4 to 2.0.5

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following tool: **tools/cami_amber**. I created this PR because I think the tool's main dependency is out of date, i.e. there is a newer version available through conda.

I have updated tools/cami_amber from version 2.0.4 to 2.0.5.

**Project home page:** https://github.com/CAMI-challenge/AMBER/releases

For any comments, queries or criticism about the bot, not related to the tool being updated in this PR, please create an issue [here](https://github.com/planemo-autoupdate/autoupdate/issues/new).

### CODE REVIEW

**Comment by bgruening** [comment]
@SantaMcCloud any idea why this fails with a python error?

**Comment by SantaMcCloud** [comment]
I will have a look on it

**Comment by SantaMcCloud** [comment]
It seems that the tool can not load the file correctly, or at least the path is not correctly split by the function. I will take a deeper look into it later and will create an issue when I found out more @bgruening

**Comment by SantaMcCloud** [comment]
So I just review the code and push and open an RP https://github.com/CAMI-challenge/AMBER/pull/56 and Issue https://github.com/CAMI-challenge/AMBER/issues/57 to fix the problem. The problem was that they wrote new code but, I think, forgot about the genome_coverage file which also use the rewrote code and because of this it came to this problem.

**Comment by SantaMcCloud** [comment]
@bgruening They pushed a new version `2.0.6`to fix the problem. So after updating it, it should work fine!

**Comment by bgruening** [comment]
Awesome, thanks for following up upstream. If you like link the issue here for traceability. 

I will wait for the conda bot to create a new package. Good that your tests catched this!

**Comment by SantaMcCloud** [comment]
No problem! 

Yes, i try always to use all parameter in the test to catch any problem :)

**Comment by bgruening** [comment]
The bot is not doing it, can you create a new package?

**Comment by SantaMcCloud** [comment]
You mean Update the recipe or the wrapper?

**Comment by bgruening** [comment]
The recipe is missing correct?

**Comment by SantaMcCloud** [comment]
no it is not messing and the bot also update it to the current version https://github.com/bioconda/bioconda-recipes/blob/master/recipes/cami-amber/meta.yaml

**Comment by SantaMcCloud** [comment]
i can Manually edit the wrapper here to update it to `2.0.6`or should i open a new PR?

**Comment by bgruening** [comment]
Ah ok, missed that update. Please update the file here in this PR. No new PR needed.

**Comment by SantaMcCloud** [comment]
How can I change it here? Via review?

File: tools/cami_amber/macros.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
             <yield/>
         </requirements>
     </xml>
-    <token name="@TOOL_VERSION@">2.0.4</token>
+    <token name="@TOOL_VERSION@">2.0.5</token>
```
Comment (bgruening) [inline tools/cami_amber/macros.xml:8]:
```suggestion
    <token name="@TOOL_VERSION@">2.0.6</token>
```

**Comment by bgruening** [comment]
If you don't have write access you can use github suggestions: https://haacked.com/archive/2019/06/03/suggested-changes/

I just did this. If it turns green is it good to go from your side?

**Comment by SantaMcCloud** [comment]
Ah okay i did guess that it would work like this but befor i do something stupid i wanted to ask :)

Yes it is good for me when all test are positiv!

**Comment by SantaMcCloud** [comment]
And thank you!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks a lot for fixing this properly in upstream!

**Pull request merged**